docs/upload-your-snap.md

```diff
@@ -1,69 +1,154 @@
-# Upload to the store
+---
+title: "Release your snap"
+---
 
```

Comment:
@caldav ^ Looks like this doesn't work (seehttps://github.com/dholbach/snapcraft/blob/4c63797af584a23a9027d399380cb1bbe04b8ef5/docs/upload-your-snap.md). Is this important for snapcraft.io? I copied it from there.